### DISCUSSION TITLE
How do I get added to slack channel?

### DISCUSSION THREAD

**Discussion by Solace-Studios** [arkivanov/Decompose#181] [Q&A] [answered]
How do I get added to slack channel?

**Comment by arkivanov** [comment, marked as answer]
You should join Kotlinlang Slack workspace. You can sign-up here: https://surveys.jetbrains.com/s3/kotlin-slack-sign-up. After that you will be able to find [#decompose](https://kotlinlang.slack.com/archives/C03H3N51SKT) channel there.
